### ISSUE TITLE
BPT not synching automatically

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#1060]
It's Saturday, June 24, 2017 at 10:20 a.m. as I write this.  I just added a application fee to BPT by hand for a performer and went to force the system to synch so there was no lag.  When I got to http://burlesque-expo.com/ticketing/transactions, it displayed the last synch time -- almost two weeks ago.

The ticketing system was last synchronized at Mon 12 Jun 2017 00:45:25 UTC. 

I have since synched the tickets by hand, but this shouldn't be failing, right?

**Comment by bethlakshmi** [comment]
OK, I poked it, added a line of code, ran it manually and got it to pass, then deleted a transaction and watched it sync - success.

I am not fully confident in this.. so let's keep an eye out.  Do you have more transactions to add?  If so, add them, wait 8 minutes and just go to the transaction page.   The script runs every 5 min, so that should work.

**Comment by bethlakshmi** [comment]
This now runs with the new updates to speed it up, too.

**Comment by burlexpo** [comment]
@bethlakshmi something is still hinky here.  Today is Thursday, June 29 and it's 13"38 UTC.  I just went to http://burlesque-expo.com/ticketing/transactions

and the display reads: The ticketing system was last synchronized at Sun 25 Jun 2017 19:00:03 UTC.

**Comment by bethlakshmi** [comment]
That's going to be that way.  The logic for that data point is:

- cron job runs
    - if (and only if) cron job finds a transaction
         - cron job updates the "last sync" value with the current day time.


If the job ran, but did NOT find a transaction to sync.... no date update.  

I counted the transactions in BPT and counted the transactions here, and they are exactly equal.  But I can't tell when the last BPT transaction came in... my bet would be w/in 5 minutes of Sun 25 Jun 2017 19:00:03 UTC

FYI - also - I added the Late Night Lust and Mini Expo onto our ticket listings so they show up on our site and also on the show pages!  After all - we should sell tickets now that we have prettier pages!!!
